app/src/processing/app/SerialPlotter.java

```diff
@@ -287,19 +316,81 @@ public void message(final String s) {
       if(parts.length == 0) {
         continue;
       }
-
+	
       int validParts = 0;
+      int validLabels = 0;
       for(int i = 0; i < parts.length; ++i) {
-        try {          
-          double value = Double.valueOf(parts[i]);
+        Double value = null;
+        String label = null;
+        
+        // all commands start with #
+        if(parts[i].startsWith("#") && parts[i].length() > 1) {
```

Comment:
I'm not a big fan of hardcoded commands but I appreciate the idea. 
Probably a standalone commit (like "add CLEAR and SET_SIZE runtime commands to Serial Plotter") could do the trick and make discussion easier

Reply:
Done in pull request #7461